src/util.h

```diff
@@ -572,6 +572,21 @@ enum ERecorderState
     RS_RECORDING = 3
 };
 
+inline QString csRecorderStateToString ( ERecorderState eRecorderState )
```

Comment:
I think this is not necessary. In GetTitle() just check for RS_RECORDING.

Reply:
Yes, now it's just shown or not shown  Wasn't even sure when I had three outputs.

Reply:
Removed.